x-pack/plugins/security_solution/public/overview/containers/overview_cti_links/api.ts

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import { KibanaServices } from '../../../common/lib/kibana';
+import { EPM_API_ROUTES } from '../../../../../fleet/common';
+
+export interface IntegrationResponse {
+  id: string;
+  status: string;
+  savedObject?: {
+    attributes?: {
+      installed_kibana: Array<{
+        type: string;
+        id: string;
+      }>;
+    };
+  };
+}
+
+export const fetchIntegrations = () =>
```

Comment:
Nit: `Integrations` is a little vague here (on the type as well); maybe 
```suggestion
export const fetchFleetIntegrations = () =>
```
?

Reply:
fixed